indico/modules/events/persons/blueprint.py

```diff
@@ -32,6 +32,7 @@
 
 # EventPerson operations
 _bp.add_url_rule('/persons/<int:person_id>', 'update_person', RHUpdateEventPerson, methods=('PATCH',))
+_bp.add_url_rule('/persons/<int:person_id>', 'sync_with_user', RHSyncEventPerson, methods=('POST',))
```

Comment:
```suggestion
_bp.add_url_rule('/persons/<int:person_id>/sync', 'sync_with_user', RHSyncEventPerson, methods=('POST',))
```

POST on the resource url directly is not very clear (in fact such a POST is usually to replace the whole thing) - so having some custom verb in the URL is nicer

Reply:
ahh yeah, missed that.. :+1: